src/electron/views/root-container/components/root-container.scss

```diff
@@ -7,10 +7,13 @@ body {
     margin: 0px;
     height: 100%;
     width: 100%;
-    background-color: $neutral-0;
     font-family: $fontFamily;
 }
 
+#root-container {
```

Comment:
looks like this change affects HC - running the unified app with HC on (through settings or `insightsUserConfiguration.setHighContrastMode(true)`) shows a white background after this PR on the device connect view

Reply:
I see why now! It's because these being imported using css modules, so I should be using :global here. 😄